### ISSUE TITLE
chapter 2: long kube-apiserver.service file isn't available electronically

### ISSUE THREAD

**Issue by mikeblas** [PacktPublishing/The-Kubernetes-Bible#2]
On page 39, instructions are given to supply a long `kube-apiserevr.service` file. This file is about 40 lines long, lots of complicated numbers and URLs and spacing.

I came here to get an electronic copy of the file, but itn'st available. Where can I get this file so I don't have to type it in and debug my typos?

**Comment by milliron** [comment]
Agreed. Also some of the yaml files, like the one to set up a cronjob, have errors in them.
